### PR TITLE
Test an example _run targets in CI

### LINKED ISSUE #36: Add _run targets to CI
## Expected Behavior
Test an example _run targets with a short timeout (eg 5 seconds) in CI

## Actual Behavior
Add something like `bazel run //examples:empty_fuzz_test_run --config=asan-libfuzzer -- --timeout_secs=5` to CI

### PR DESCRIPTION
**Pull request by tengpengli**
**Commit Message:**
Test an example _run targets in CI
**Additional Description:**
Test an example empty_fuzz_test_run targets with a short timeout (eg 5 seconds) in CI
**Testing:**
CI testing and local testing.
**Docs Changes:** N/A
**Fixes #36**

Signed-off-by: tengpeng <tengpeng.li2020@gmail.com>

	modified:   .github/workflows/bazel_test.yml

### CODE REVIEW

**Comment by tengpengli** [comment]
Have problem installing the absl-py. Since this is not in the critical path, I will pend this issue for a little while and come back to fix this later

File: .github/workflows/bazel_test.yml [hunk 1]
```diff
@@ -31,3 +35,4 @@ jobs:
         # The "<fuzzer/FDP...>" is only supported since clang-10 while the clang version in github runner is 9
         run: |
           bazel build --verbose_failures --build_tag_filters=fuzz_test,-FDP --config=asan-libfuzzer //examples/...
+          bazel run //examples:empty_fuzz_test_run --config=asan-libfuzzer -- --timeout_secs=5
```
Comment (tengpengli) [inline .github/workflows/bazel_test.yml:38]: What version of python is used when running _run?

File: .github/workflows/bazel_test.yml [hunk 2]
```diff
@@ -23,6 +23,10 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Install deps 
+        run: |
+          pip install wheel absl-py
+          pip3 install wheel absl-py
```
Comment (tengpengli) [inline .github/workflows/bazel_test.yml:29]: Check the version of pip and see if it's the same version as python

File: .github/workflows/bazel_test.yml (hunk 1 above)
Comment (tengpengli) [reply .github/workflows/bazel_test.yml:39]: The `python` command in github runner is python2. I have to declare `alias python="python3"` to run python3

File: .github/workflows/bazel_test.yml (hunk 2 above)
Comment (tengpengli) [reply .github/workflows/bazel_test.yml:29]: Default `pip` is `pip2`.

**Comment by tengpengli** [comment]
The codes are ready to serve. PTAL!

File: .github/workflows/bazel_test.yml [hunk 3]
```diff
@@ -23,11 +23,17 @@ jobs:
...
       - name: Run all tests
         working-directory: ./
         # Build //examples/...
         # Add "bazel test --config=clang //... --test_tag_filters=test"in the future if any test rules are added
         # The -FDP is for filtering out the files which contains "#include <fuzzer/FuzzedDataProvider.h>",
         # The "<fuzzer/FDP...>" is only supported since clang-10 while the clang version in github runner is 9
         run: |
+          alias python='python3'
           bazel build --verbose_failures --build_tag_filters=fuzz_test,-FDP --config=asan-libfuzzer //examples/...
+          bazel run //examples:empty_fuzz_test_run --config=asan-libfuzzer -- --timeout_secs=5
```
Comment (stefanbucur) [inline .github/workflows/bazel_test.yml:39]: It's great to test this command; I was wondering what we can do here to make the test fail if something other than a build goes wrong. Some failure scenarios in mind:

* The timeout is not obeyed, causing the test to run indefinitely. How will we catch this error? Can we make the command fail automatically (say if we leave it to run for 5 seconds, forcefully terminate it after 30 seconds) so it doesn't take too long to run? @asraa did Envoy have to come up with a solution to this problem?
* The command runs, but doesn't produce the expected output. For example, if we run a crashing fuzz target, running that command should expose the crash.

For the second problem, I imagine writing separate shell scripts and invoking them from here seems more scalable.

In any case, we can still push what we have now, and open a separate issue for improving it.

Comment (asraa) [reply .github/workflows/bazel_test.yml:39]: We could add a timeout to the test script as a whole (like 5-10 min), which won't give fine-grained control over each test runtime, but would catch something like a fuzzer is running indefinitely 
(https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes)

For the second, I agree, having a framework to say "run this, expect with grep some error line comes up" would be cool

**Pull request merged**